app/api/helpers/order.py

```diff
@@ -73,9 +73,10 @@ def create_pdf_tickets_for_holder(order):
             save_to_db(holder)
 
         # create order invoices pdf
-        order_ticket_info = OrderTicket.query.filter_by(order_id=order.id).one()
+        order_tickets = OrderTicket.query.filter_by(order_id=order.id).all()
+
         create_save_pdf(render_template('pdf/order_invoice.html', order=order, event=order.event,
-                        tax=order.event.tax, tickets=order.tickets, order_tickets_info=order_ticket_info),
+                        tax=order.event.tax, tickets=order.tickets, order_tickets=order_tickets),
```

Comment:
Can we now remove order.tickets if it is not used anywhere as it is an extra query

Reply:
@iamareebjamal Yup. It isn't being used anymore.